src/renderer/components/dock/upgrade-chart/view.tsx

```diff
@@ -96,7 +98,7 @@ export class NonInjectedUpgradeChart extends React.Component<UpgradeChartProps &
     const versions = await helmChartStore.getVersions(release.getChart());
 
     this.versions.replace(versions);
-    this.version = this.versions[0];
+    this.version = first(this.versions);
```

Comment:
Note: had to do this since opening tab for the purpose of upgrading chart shows warning without the data. There is still open issue for the crashes in upgrade #5852.

Reply:
What is the difference then? Or do you mean a mobx warning in the console?